pandas/tests/io/test_pickle.py

```diff
@@ -85,6 +85,13 @@ def compare_element(result, expected, typ):
 legacy_dirname = os.path.join(os.path.dirname(__file__), "data", "legacy_pickle")
 files = glob.glob(os.path.join(legacy_dirname, "*", "*.pickle"))
 
+if not len(files):
+    # Probably is a wheel build(no data files), so skip the entire module
+    pytest.skip(
+        "Could not find legacy pickle files, skipping test_pickle.py tests",
```

Comment:
Looks like not all pickle tests require this pickle file so we shouldn't skip the the entire module. 

Since this and `legacy_pickle` is only used once in `test_pickles` can we inline all this code and skip that test directly?

Reply:
Reverting #48924 is probably the easiest option here.
I could move the glob inside of the test, but then the test wouldn't be parameterized anymore.

Reply:
I think in this case it's okay to forgo the parameterization and just put the glob in `test_pickles`. I am a little worried about `empty_parameter_set_mark = "skip"` wrongly skipping tests globally